airflow/www/views.py

```diff
@@ -94,7 +94,16 @@
 
 UTF8_READER = codecs.getreader('utf-8')
 
-dagbag = models.DagBag(settings.DAGS_FOLDER, store_serialized_dags=STORE_SERIALIZED_DAGS)
+try:
+    async_dagbag_loader = conf.getboolean('webserver', 'async_dagbag_loader')
```

Comment:
This is not a "clean-up" -- it is a whole new feature.

And it was a feature that was discussed and rejected in the original serialized DAG proposal/PR. Please to not bring it back in without a very clear indication of what problem it solves over the current method against  1.10.10.

Reply:
Sorry for that. I thought it got lost or was removed. It is not necessary with dag serialization and definitely such big functionalities should be first discussed via. AIP